app/src/androidTest/java/com/courseproject/tindar/ds/DatabaseHelperTest.java

```diff
@@ -54,6 +56,7 @@ public void setUp() {
 
     @After
     public void tearDown() {
+        dbHelper.deleteAllAccounts();
```

Comment:
Doesn't this ruin persistence? Making an issue for this. https://github.com/CSC207-2023Y-UofT/course-project-tindar/issues/77